app/src/main/kotlin/com/intive/picover/main/navigation/view/PicoverNavHost.kt

```diff
@@ -22,13 +25,22 @@ fun PicoverNavHost(
 		startDestination = startDestination,
 	) {
 		composable("parties") {
-			PartiesScreen()
+			PartiesScreen(
+				viewModel = hiltViewModel(),
+				navController = navController,
+			)
 		}
 		composable("camera") {
 			CameraScreen(hiltViewModel())
 		}
 		composable("profile") {
 			ProfileScreen()
 		}
+		composable(
+			route = "partyDetails/{partyId}",
+			arguments = listOf(navArgument("partyId") { type = NavType.IntType }),
```

Comment:
I think it would be better to extract argument keys to vals, but what do others think?

extracting args keys -> 👍 
harcoding args keys -> 👎

Reply:
#76